.clang-format

```diff
@@ -0,0 +1,25 @@
+BasedOnStyle: LLVM
+IndentWidth:  2
+ColumnLimit: 80
+
+PointerAlignment: Left
+ReferenceAlignment: Left
+
+BreakTemplateDeclarations: Yes 
+IndentRequiresClause: true
+
+AlignConsecutiveAssignments: Consecutive
+AlignConsecutiveDeclarations: Consecutive
+AlignTrailingComments: true
+SpacesBeforeTrailingComments: 2
+
+KeepEmptyLines:
+  AtEndOfFile: true
+  AtStartOfBlock: true
+  AtStartOfFile: false
+MaxEmptyLinesToKeep: 1
+
+NamespaceIndentation: Inner
+
+IncludeBlocks: Preserve
+
```

Comment:
Does this file require `\n\n` at the end? If so then can we remove the stray blank line?

```suggestion
```